app/models/program.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+class Program
+  def self.all
```

Comment:
What do you think about adding `class << self` so we don't have `self.` on all the methods. Just a personal preference, keeps things a bit easier to read for me, but I don't feel strongly about it.

Reply:
I don't love this syntax but have updated to match 👌